qt/widgets/common/inc/MantidQtWidgets/Common/FitPropertyBrowser.h

```diff
@@ -321,6 +321,10 @@ class EXPORT_OPT_MANTIDQT_COMMON FitPropertyBrowser
   double getPeakHeightOf(const QString &prefix);
   void setPeakFwhmOf(const QString &prefix, double value);
   double getPeakFwhmOf(const QString &prefix);
+  std::string getWidthParameterNameOf(const QString &prefix);
+  std::string getCentreParameterNameOf(const QString &prefix);
+  bool getParameterNameExplicitlySetOf(const QString &prefix,
```

Comment:
Open for discussion, but maybe this should be called `isParameterNameExplicitlySet` or  `isParameterExplicitlySet`, the current name `getParameterNameExplicitlySet` makes it seem like its returning a parameter or parameter name.